src/ios/updateVersion.sh

```diff
@@ -18,7 +18,8 @@ curl -Ls https://github.com/mixpanel/mixpanel-iphone/archive/$ZIPFILE -O $ZIPFIL
 unzip $ZIPFILE
 rm -f $ZIPFILE
 mv Mixpanel Mixpanel_bak
-mv mixpanel-iphone-$VERSION/Mixpanel .
+mv mixpanel-iphone-$VERSION/Sources .
+mv Sources Mixpanel
```

Comment:
suggestion: combine into 1 line :)
```suggestion
mv mixpanel-iphone-$VERSION/Sources ./Mixpanel
```